pkg/vtpm/src/server.cpp

```diff
@@ -340,13 +352,19 @@ parseRequest(int sock,
 int
 handleRequest (int sock, google::protobuf::uint32 size)
 {
-    char payload[size+CODED_STRM_HDR_LEN];
     eve_tools::EveTPMRequest request;
     eve_tools::EveTPMResponse response;
     int byteCnt = 0, rc = success;
-    const char *cmdArgs[MAX_ARGS+1];
+    const char *cmdArgs[MAX_ARGS+1] = {};
```

Comment:
Leaky pointer, we are not releasing resources

Reply:
this is allocated on stack, isn't it?

Reply:
then why do we free it on 476

Reply:
it is `char**` array allocated on the stack, I allocated/free `char*` memory that fills the array indexes.

Reply:
got it, thanks